src/pages/MinerDashboard/Header/Stats.tsx

```diff
@@ -107,7 +107,7 @@ const BalanceProgressBar: React.FC<{
             <PayoutNumber>
               {dateUtils.formatDistance(
                 addSeconds(new Date(), payoutInSeconds)
-              )}
+              ).toString().substring(3)}
```

Comment:
This method is not compatible with other languages and is not a best practice. Can you please just remove `in` from the non-green text?